docs/src/installation.md

```diff
@@ -1,63 +1,100 @@
-| [MOSEK](http://www.mosek.com/)                                                   | [Mosek.jl](https://github.com/JuliaOpt/Mosek.jl)                                | `MosekSolver()`           |  Comm.      | X     | X     | X     | X     |       | X     |
-| [NLopt](http://ab-initio.mit.edu/wiki/index.php/NLopt)                           | [NLopt.jl](https://github.com/JuliaOpt/NLopt.jl)                                | `NLoptSolver()`           |  LGPL       |       |       |       | X     |       |       |
-| [SCS](https://github.com/cvxgrp/scs>)                                            | [SCS.jl](https://github.com/JuliaOpt/SCS.jl)                                    | `SCSSolver()`             |  MIT        | X     | X     |       |       |       | X     |
+JuMP depends on solvers to solve optimization problems. Most solvers are not
+written in Julia, and some require commercial licenses to use, so installation
+is often more complex. We list below the currently available solvers.
+
+!!! note
+    This list is open for new contributions. See also
+    [Interacting with solvers](@ref) and the
+    [MathOptInterface docs](http://www.juliaopt.org/MathOptInterface.jl/v0.6.1/)
+    for more details on how JuMP interacts with solvers. Please get in touch
+    with any questions about connecting new solvers with JuMP.
+
+
+| Solver                                                                         | Julia Package                                       | License | Supports                    |
+| ------------------------------------------------------------------------------ | --------------------------------------------------- | ------- | --------------------------- |
+| [Artelys Knitro](http://artelys.com/en/optimization-tools/knitro)              | [KNITRO.jl](https://github.com/JuliaOpt/KNITRO.jl)  | Comm.   | NLP, MINLP                  |
+| [BARON](http://archimedes.cheme.cmu.edu/?q=baron>)                             | [BARON.jl](https://github.com/joehuchette/BARON.jl) | Comm.   | MINLP                       |
+| [Cbc](https://projects.coin-or.org/Cbc)                                        | [Cbc.jl](https://github.com/JuliaOpt/Cbc.jl)        | EPL     | MILP                        |
+| [Clp](https://projects.coin-or.org/Clp)                                        | [Clp.jl](https://github.com/JuliaOpt/Clp.jl)        | EPL     | LP                          |
+| [CPLEX](http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/) | [CPLEX.jl](https://github.com/JuliaOpt/CPLEX.jl)    | Comm.   | LP, MILP, SOCP, MISOCP      |
+| [CSDP](https://projects.coin-or.org/Csdp/)                                     | [CSDP.jl](https://github.com/JuliaOpt/CSDP.jl)      | EPL     | SDP                         |
+| [ECOS](https://github.com/ifa-ethz/ecos)                                       | [ECOS.jl](https://github.com/JuliaOpt/ECOS.jl)      | GPL     | SOCP                        |
```

Comment:
What is the reasoning behind the "LP". ECOS and CSDP can also do LP even if they might not be competitive at it. Does "LP" means "has a simplex solver" ? Or do we mean "should consider using this solver for a pure LP problem" ? Here the heading says "supports".

Reply:
LP should be listed if the solver supports it. This isn't the right place to make judgments on whether the solver is competitive.

Reply:
Which solver does not support LP then ?

Reply:
Fixed. The only solver that I didn't say LP for was Cbc because it doesn't return duals so it can't prove that it solved the LP :)

Reply:
It makes sense to only call LP for solvers with duals